config/routes.rb

```diff
@@ -0,0 +1,15 @@
+
+
+Rails.application.routes.draw do
+  #   resources :greetings
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  namespace :api do
+    namespace :v1 do
+      resources :greetings
+    end
```

Comment:
Please create only one route which receives a request from the client for a random greeting. Kindly don't need to expose all RESTful routes.